subprojects/language-native/src/main/java/org/gradle/language/swift/tasks/SwiftCompile.java

```diff
@@ -240,6 +243,16 @@ public ConfigurableFileCollection getModules() {
         return modules;
     }
 
+    /**
+     * The Swift language version support for compiling the source.
+     *
+     * @since 4.5
+     */
+    @Input
+    public Property<SwiftLanguageVersion> getSwiftLanguageVersionSupport() {
```

Comment:
Update the name and comment, to match the property on `SwiftComponent` (if it changes).

Reply:
Done.